CLI subsample improvements: (#1925)
CloudCompare/CloudCompare: CHANGELOG.md

		- New sub-option for the -SF_ARITHMETIC command: -IN_PLACE, to update the scalar field in place, without creating a new SF
		- Most methods using scalar fields as input will now also accept the scalar field name (in lieu of the SF index)
		- New sub-option for -APPLY_TRANS command: -INVERSE to inverse the transformation matrix before it is applied.

> 		- New sub-option for -SS OCTREE command: NUMBER_OF_POINTS {number} to subsample with the highest octree number where the the resulting point count won't exceed the given number of points
> 		- New sub-option for -SS OCTREE command: CELL_SIZE {size} to calculate octree number from bounding box, and the given cell_size
> 		- New sub-option for -SS RANDOM command: PERCENT {number} to calculate count from PERCENT. PERCENT should be a decimal number between 0 and 100.
> 		- New sub-option for -SS OCTREE NUMBER_OF_POINTS command: PERCENT {number} to calculate NUMBER_OF_POINTS from PERCENT. PERCENT should be a decimal number between 0 and 100.

Reviewer: ```suggestion
		- New sub-options for -SS command:
				* -SS OCTREE NUMBER_OF_POINTS {number} to subsample with the highest octree level for which the resulting point count won't exceed the given number of points
				* -SS OCTREE NUMBER_OF_POINTS PERCENT {number} to calculate NUMBER_OF_POINTS from PERCENT. PERCENT should be a decimal number between 0 and 100.
				* -SS OCTREE CELL_SIZE {size} to deduce the octree level from the given cell size.
				* -SS RANDOM PERCENT {number} to calculate the number of sampled points from PERCENT. PERCENT should be a decimal number between 0 and 100.
```
